helpers/parse_args.go

```diff
@@ -83,12 +88,17 @@ func ParseArgs() error {
 	if os.Getenv("AZURE_SAMPLES_KEEP_RESOURCES") == "1" {
 		keepResources = true
 	}
-	armEndpointString = os.Getenv("AZURE_ARM_ENDPOINT")
+
+	envName = os.Getenv("AZURE_ENVIRONMENT")
```

Comment:
I'm thinking that we should define these well-known environment vars as exported constants in go-autorest.  What do you think?

Reply:
I would agree

Reply:
I think we should go a step further and create a [viper](https://github.com/spf13/viper) instance in go-autorest that looks for them in a number of places, including environment variables. 🐍